receiver/prometheusreceiver/internal/transaction_test.go

```diff
@@ -365,6 +572,11 @@ func TestMetricBuilderGauges(t *testing.T) {
 				pt0.SetStartTimestamp(0)
 				pt0.SetTimestamp(tsNanos)
 				pt0.Attributes().PutString("foo", "bar")
+				e0 := pt0.Exemplars().AppendEmpty()
+				e0.SetTimestamp(pcommon.NewTimestampFromTime(time.UnixMilli(1663113420863)))
+				e0.SetDoubleVal(1)
+				e0.FilteredAttributes().PutString("foo", "bar")
+				e0.SetTraceID(getTraceIdFromString("4ff62c778aa548b433069a6bba8ffc23"))
```

Comment:
**NEVER** unittest with the same code. Here in `getTraceIdFromString` you have a duplicate of the logic inside family on how to parse the trace id, and you duplicate it. Use pre-calculated values and expected values.

Reply:
Done.. using byte array instead